docs/querying/query-context.md

```diff
@@ -37,34 +37,34 @@ Note that setting query context will override both the default value and the run
 
 Unless otherwise noted, the following parameters apply to all query types.
```

Comment:
```suggestion
Unless otherwise noted, the following properties apply to all query types.
```
If they're properties in the table, should they be properties in the intro?

Reply:
Apparently "parameters" is all over this doc but I think "properties" would be the more accurate term. Updating.